app/Http/Controllers/AdminDashboardController.php

```diff
@@ -19,8 +19,9 @@ class AdminDashboardController extends Controller
     public function dashboard(): View|Factory|Application
     {
         $articles = Article::orderBy('updated_at', 'desc' ,'created_at', 'desc')->Paginate(10);
+        $categories = Article::find(1)->categories()->get();
```

Comment:
@DJMasaru 

Articleの一覧 with Category なので、ここはwithを使ってeager loadingすることで、リレーションを全部foreachで回して取ってくるとかやらなくて良くなるよ

```suggestion
        $categories = Article::with(['categories'])->get();
```